stemcell_builder/stages/system_azure_init/apply.sh

```diff
@@ -5,7 +5,7 @@ set -e
 base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
 
-packages="python3 python3-pyasn1 python3-setuptools"
+packages="python3 python3-pyasn1 python3-setuptools python-is-python3 cloud-init"
```

Comment:
discussed about the `python-is-python3` package
and if it should be installed globally on all stemcells as python3 is installed by default on all stemcells.
but as azure is the only one using python in our stemcells anyways it seems to make sense to put it here